src/chrome/content/rules/PirateParty.xml

```diff
@@ -38,46 +24,75 @@ Fetch error: http://europeanpirates.eu/ => http://europeanpirates.eu/: (60, 'SSL
 <ruleset name="Pirate Party" default_off='failed ruleset test'>
```

Comment:
Note that this ruleset is default disabled because it failed the fetch test. If your changes resolve those issues, remove the default_off and the comment at the top of the file.